modules/about-dynamic-plug-ins.adoc

```diff
@@ -0,0 +1,34 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/web_console/dynamic-plug-ins.adoc
+
+:_content-type: CONCEPT
+[id="about-dynamic-plugins_{context}"]
+= About dynamic plug-ins
+
+A dynamic plug-in allows you to add custom pages and other extensions to your interface at runtime. Plug-ins are registered with the console using the `ConsolePlugin` custom resource, and they are enabled in the `console-operator` configuration by a cluster administrator.
```

Comment:
```suggestion
A dynamic plug-in allows you to add custom pages and other extensions to your interface at runtime. The `ConsolePlugin` custom resource registers plug-ins with the console, and a cluster administrator enables plug-ins in the `console-operator` configuration.
```
Just a suggestion if you want to avoid passive voice.